x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearningFeatureSet.java

```diff
@@ -215,24 +219,29 @@ private void addJobsUsage(GetJobsStatsAction.Response response) {
                         js -> new StatsAccumulator()).add(detectorsCount);
                 modelSizeStatsByState.computeIfAbsent(jobState,
                         js -> new StatsAccumulator()).add(modelSize);
+                forecastStatsByState.computeIfAbsent(jobState,
+                        js -> new ForecastStats()).merge(jobStats.getForecastStats());
```

Comment:
not sure if I get the question, only in case the key does not exist yet it is equivalent to `jobStats.getForecastStats()`. I think I can simplify this using the `Map::merge` function in this case.